core/storage/suggestion/gae_models.py

```diff
@@ -43,13 +43,20 @@
 STATUS_ACCEPTED: Final = 'accepted'
 STATUS_IN_REVIEW: Final = 'review'
 STATUS_REJECTED: Final = 'rejected'
+STATUS_ACCEPTED_WITH_EDITS: Final = 'accepted_with_edits'
```

Comment:
I advise that you don't call these statuses. This is going to confuse things. Instead you should use REVIEW_OUTCOME_ACCEPTED, REVIEW_OUTCOME_ACCEPTED_WITH_EDITS, etc. 

Otherwise, with what you have, it leads to questions about why STATUS_CHOICES doesn't include STATUS_ACCEPTED_WITH_EDITS etc.

Reply:
Changed.